typings/package.json

```
@@ -4,7 +4,7 @@
     "access": "public"
   },
   "author": "Blockchains Inc",
-  "version": "0.0.3",
+  "version": "0.0.5",
```

```suggestion
  "version": "0.0.5",
  "license": "Apache-2.0",
```

can we add a license, to make black duck happy

---

updated. ^^